src/zkevm_specs/evm/instruction.py

```diff
@@ -280,28 +284,33 @@ def rlc_to_int_exact(self, rlc: RLC, n_bytes: int) -> int:
 
     def word_to_lo_hi(self, word: RLC) -> Tuple[Sequence[int], Sequence[int]]:
         word_le_bytes = self.rlc_to_le_bytes(word)
-        assert len(word_le_bytes) == 32, "Expected RLC to contain 32 bytes"
-        return self.bytes_to_int(word_le_bytes[:16]), self.bytes_to_int(word_le_bytes[16:])
+        assert len(word_le_bytes) == 32, "Expected word to contain 32 bytes"
+        return self.bytes_to_fq(word_le_bytes[:16]), self.bytes_to_fq(word_le_bytes[16:])
 
     def int_to_rlc(self, value: int, n_bytes: int) -> RLC:
         return RLC(value, self.randomness, n_bytes)
 
     def bytes_to_int(self, value: Sequence[int]) -> int:
         assert len(value) <= MAX_N_BYTES, "Too many bytes to composite an integer in field"
-
         return int.from_bytes(value, "little")
 
-    def range_lookup(self, value: int, range: int):
+    def bytes_to_fq(self, value: Sequence[int]) -> FQ:
```

Comment:
should this be bytes?
```suggestion
    def bytes_to_fq(self, value: bytes) -> FQ:
```

Reply:
Updated